includes/wikia/WikiaPageType.class.php

```diff
@@ -264,20 +265,22 @@ public static function getMainPageName() {
 	/**
 	 * Check if current page is home page
 	 *
+	 * @param int|null $wikiId
 	 * @return bool
 	 */
-	public static function isWikiaHomePage() {
-		global $wgEnableWikiaHomePageExt;
+	public static function isWikiaHomePage( $wikiId = null ) {
+		global $wgCityId;
 
-		return !empty( $wgEnableWikiaHomePageExt );
+		return WikiFactory::getVarValueByName( 'wgEnableWikiaHomePageExt', $wikiId ?? $wgCityId );
 	}
 
 	/**
 	 * Check if current page is corporate page
 	 *
+	 * @param int|null $wikiId
 	 * @return bool
 	 */
-	public static function isCorporatePage() {
-		return self::isWikiaHub() || self::isWikiaHomePage();
+	public static function isCorporatePage( $wikiId = null ) {
+		return self::isWikiaHub( $wikiId ) || self::isWikiaHomePage( $wikiId );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)